docs/source/pillars/information_governance.md

```diff
@@ -91,6 +91,14 @@ _The ability of the TRE organisation to monitor compliance with internal and ext
 * - You should report on and share outcomes of each audit of your TRE organisation with the required bodies.
   - This may be a requirement of continued accreditation by external organisations or regulatory bodies.
   - Recommended
+* - You log data access queries and publish these logs
+  - Transparency is a key principle for an environment that is trusted for research.
```

Comment:
Maybe, I'm not sure. If it is, shouldn't we state this somewhere more prominently and shouldn't it be a more common theme in this document?

I would think this depends very strongly on the source of the data and nature of the research.

Reply:
I would agree with this!

Reply:
Yes to transparency about the aims of the research, maybe/no to transparency about the exact details of everything a researcher is doing (see points above).